src/CountUp.jsx

```diff
@@ -39,6 +39,15 @@ class CountUp extends React.Component {
     this.startAnimation();
   }
 
+  shouldComponentUpdate(nextProps){
+    const shouldNotUpdate = this.isEquivalent(nextProps,this.props);
+
+    if(nextProps.redraw){
+      return true;
+    }
+    return !shouldNotUpdate;
+  }
+
```

Comment:
I think using the react [shallow compare helper function](https://facebook.github.io/react/docs/shallow-compare.html) would be a better idea.
Could you `npm install -D react-addons-shallow-compare`, import it in this file and then replace this block with like following:

```js
shouldComponentUpdate = (nextProps, nextState) => (nextProps.redraw
  ? true
  : shallowCompare(this, nextProps, nextState)
)
```